fastlane/framework.rb

```diff
@@ -36,43 +39,58 @@ def self.all
+        frameworks["wire-ios-notification-engine"].add_dependency(frameworks["WireLogging"])
+
         frameworks["wire-ios-sync-engine"].add_dependency(frameworks["wire-ios-request-strategy"])
         frameworks["wire-ios-sync-engine"].add_dependency(frameworks["WireAPI"])
-        frameworks["wire-ios-sync-engine"].add_dependency(frameworks["WireDomain"])
         frameworks["wire-ios-sync-engine"].add_dependency(frameworks["WireAnalytics"])
+        frameworks["wire-ios-sync-engine"].add_dependency(frameworks["WireDomain"])
+        frameworks["wire-ios-sync-engine"].add_dependency(frameworks["WireLogging"])
+
         frameworks["wire-ios-share-engine"].add_dependency(frameworks["wire-ios-request-strategy"])
+        frameworks["wire-ios-share-engine"].add_dependency(frameworks["WireLogging"])
 
         frameworks["wire-ios-request-strategy"].add_dependency(frameworks["wire-ios-data-model"])
         frameworks["wire-ios-request-strategy"].add_dependency(frameworks["WireAPI"])
-        
+        frameworks["wire-ios-request-strategy"].add_dependency(frameworks["WireLogging"])
+
         frameworks["wire-ios-data-model"].add_dependency(frameworks["wire-ios-cryptobox"])
         frameworks["wire-ios-data-model"].add_dependency(frameworks["wire-ios-images"])
         frameworks["wire-ios-data-model"].add_dependency(frameworks["wire-ios-link-preview"])
         frameworks["wire-ios-data-model"].add_dependency(frameworks["wire-ios-protos"])
         frameworks["wire-ios-data-model"].add_dependency(frameworks["wire-ios-transport"])
         frameworks["wire-ios-data-model"].add_dependency(frameworks["WireFoundation"])
-        
+        frameworks["wire-ios-data-model"].add_dependency(frameworks["WireLogging"])
+
         frameworks["wire-ios-mocktransport"].add_dependency(frameworks["wire-ios-testing"])
         frameworks["wire-ios-mocktransport"].add_dependency(frameworks["wire-ios-cryptobox"])
         frameworks["wire-ios-mocktransport"].add_dependency(frameworks["wire-ios-protos"])
 
         frameworks["wire-ios-cryptobox"].add_dependency(frameworks["wire-ios-utilities"])
+
         frameworks["wire-ios-transport"].add_dependency(frameworks["wire-ios-utilities"])
+        frameworks["wire-ios-transport"].add_dependency(frameworks["WireLogging"])
+
         frameworks["wire-ios-link-preview"].add_dependency(frameworks["wire-ios-utilities"])
+
         frameworks["wire-ios-images"].add_dependency(frameworks["wire-ios-utilities"])
 
         frameworks["wire-ios-utilities"].add_dependency(frameworks["wire-ios-system"])
         frameworks["wire-ios-utilities"].add_dependency(frameworks["WireFoundation"])
-        
+        frameworks["wire-ios-utilities"].add_dependency(frameworks["WireLogging"])
+
         frameworks["wire-ios-testing"].add_dependency(frameworks["wire-ios-system"])
-        
+
+        frameworks["wire-ios-system"].add_dependency(frameworks["WireLogging"])
+
```

Comment:
need to update also the to_scheme to return WireLogging

Reply:
done